articles/active-directory/roles/permissions-reference.md

```diff
@@ -1013,7 +1013,7 @@ Users with this role have access to all administrative features in Azure Active
 > | microsoft.directory/crossTenantAccessPolicy/partners/b2bDirectConnect/update | Update Azure AD B2B direct connect settings of cross-tenant access policy for partners |
 > | microsoft.directory/crossTenantAccessPolicy/partners/crossCloudMeetings/update | Update cross-cloud Teams meeting settings of cross-tenant access policy for partners |
 > | microsoft.directory/crossTenantAccessPolicy/partners/tenantRestrictions/update | Update tenant restrictions of cross-tenant access policy for partners |
-> | microsoft.directory/privilegedIdentityManagement/allProperties/read | Read all resources in Privileged Identity Management |
+> | microsoft.directory/privilegedIdentityManagement/allProperties/allTasks | Create and delete all resources, and read and update standard properties in Privileged Identity Management |
```

Comment:
Hi @chkoide
Thanks for your pull request. The permissions in this doc are auto generated and match what is currently in the portal, so I'm not able to approve this change. I've started a discussion with the team to confirm these permissions.
```suggestion
> | microsoft.directory/privilegedIdentityManagement/allProperties/read | Read all resources in Privileged Identity Management |
```
thanks